src/Symfony/Component/Form/Extension/Core/DataTransformer/DateTimeImmutableToDateTimeTransformer.php

```diff
@@ -18,6 +18,8 @@
  * Transforms between a DateTimeImmutable object and a DateTime object.
  *
  * @author Valentin Udaltsov <udaltsov.valentin@gmail.com>
+ *
+ * @implements DataTransformerInterface<\DateTimeImmutable|null, \DateTime|null>
```

Comment:
no need of `|null` here. The interface already turns those types into nullable ones